website/notifications/events/signals.py

```diff
@@ -0,0 +1,4 @@
+import blinker
+
+signals = blinker.Namespace()
+file_updated = signals.signal('file_updated')
```

Comment:
This doesn't belong here. Signals should go in a `signals.py` file in the module where it's used; in this case, `website.addons.base`